src/cli/argument-parser.js

```diff
@@ -113,7 +125,19 @@ export default class CLIArgumentParser {
         return true;
     }
 
-    _parseFilteringOptions () {
+    async _parseFilteringOptions () {
+        if (this.opts.testGrep)
+            this.opts.testGrep = CLIArgumentParser._optionValueToRegExp('--test-grep', this.opts.testGrep);
```

Comment:
It looks like you broke `testGrep` convertion to reqular expression when the option is located in configuration file.